app/pages/routeDetails/unloading/AddPlaceOfUnloadingPage.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.routeDetails.unloading
+
+import controllers.routeDetails.unloading.routes
+import models.{Mode, UserAnswers}
+import pages.QuestionPage
+import pages.sections.UnloadingSection
+import play.api.libs.json.JsPath
+import play.api.mvc.Call
+
+case object AddPlaceOfUnloadingPage extends QuestionPage[Boolean] {
+
+  override def path: JsPath = UnloadingSection.path \ toString
```

Comment:
I would advise putting this in `RouteDetailsSection`. The reason being that the cleanup logic can then do a `.remove(UnloadingSection)`

Reply:
Alright, sorted.